### PR TITLE
add test for modules managed command

### PR DESCRIPTION
**Pull request by anoipm**
**Description**

Changes proposed in this pull request:

- add test for modules managed command

**Related issue(s)**
See also #2117

### CODE REVIEW

File: internal/communitymodules/modules_test.go [hunk 1]
```diff
@@ -86,9 +94,125 @@ func Test_modulesCatalog(t *testing.T) {
 	})
 }
 
+func Test_ManagedModules(t *testing.T) {
+	t.Run("ok", func(t *testing.T) {
+		expectedResult := moduleMap{
+			"module1": row{
+				Name:    "module1",
+				Managed: "Managed",
```
Comment (MichalKalke) [inline internal/communitymodules/modules_test.go:102]: Please change it to `True` as the new value of the `MANAGED` field

File: internal/communitymodules/modules_test.go [hunk 2]
```diff
@@ -86,9 +94,125 @@ func Test_modulesCatalog(t *testing.T) {
...
+			},
+			"module2": row{
+				Name:    "module2",
+				Managed: "Managed",
```
Comment (MichalKalke) [inline internal/communitymodules/modules_test.go:106]: As above

File: internal/communitymodules/modules_test.go [hunk 3]
```diff
@@ -86,9 +94,125 @@ func Test_modulesCatalog(t *testing.T) {
...
+			},
+			"module3": row{
+				Name:    "module3",
+				Managed: "Managed",
```
Comment (MichalKalke) [inline internal/communitymodules/modules_test.go:110]: As above

File: internal/communitymodules/modules_test.go (hunk 1 above)
Comment (anoipm) [reply internal/communitymodules/modules_test.go]: done

File: internal/communitymodules/modules_test.go (hunk 2 above)
Comment (anoipm) [reply internal/communitymodules/modules_test.go]: done

File: internal/communitymodules/modules_test.go (hunk 3 above)
Comment (anoipm) [reply internal/communitymodules/modules_test.go]: done

**Review by MichalKalke — APPROVED** [review]

**Pull request merged**